lite/examples/model_personalization/android/app/src/main/java/org/tensorflow/lite/examples/transfer/CameraFragment.java

```diff
@@ -173,7 +173,13 @@ private void startCamera() {
           // We don't perform inference when adding samples, since we should be in capture mode
           // at the time, so the inference results are not actually displayed.
           inferenceBenchmark.startStage(imageId, "predict");
-          Prediction[] predictions = tlModel.predict(rgbImage);
+          Prediction[] predictions = null;
+          try {
+            predictions = tlModel.predict(rgbImage, modelImageSize, modelImageSize);
+          }
+          catch (Exception e){
```

Comment:
1) Could you specifically catch `NullPointerException` instead? It would be bad to catch all exceptions and ignore them in case something else happens.
2) Also, please add a comment explaining why the `NullPointerException` can be thrown and why we are ignoring it.

Thanks for your contribution!